app/docs/form-engine-client-setup/sections/quickbase.section.html

```diff
@@ -0,0 +1,27 @@
+<doc-section>
+  <doc-header id="quickbase">Quickbase</doc-header>
+  <doc-caption>Actions required to be performed on quickbase</doc-caption>
+
+  <h5>Copy Application</h5>
+    <ul>
+      <li>
+        Go to Settings under the application name
+      </li>
+      <li>Go to App Management</li>
+      <li>Copy app</li>
+        <ul>
+          <li>Data: select copy this application WITHOUT data</li>
+          <li>Users & Roles: Copy users in their roles</li>
```

Comment:
We actually don't want to copy the users in their roles, because we have a lot of internal users attached that we'd then have to delete. Instead, copy without users. Then we would recreate the "QuickStart" role and set up the QuickStart User in the copied instance with that role.